### ISSUE TITLE
Insufficient Color Contrast on visible field labels (Preferred name, Language, etc)

### ISSUE THREAD

**Issue by pdpinch** [mitodl/micromasters#1682]
> The color combination #b3b3b3/#ffffff has a contrast ratio of 2.10, which is not sufficient.
> Relevant code:
`<div style="position: absolute; opacity: 1; color: rgba(0, 0, 0, 0.298039); transition: all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms; bottom: 12px;">Select Program</div>`

Use http://contrastchecker.com/ to check contrast. Our goal is AA support.

**Comment by roberthouse54** [comment]
So presumably, this github page would fail the same test, for the comment field text that says "Leave a comment" in light gray? This is pretty standard across the web... I am a bit surprised at this. The comment here is about the field placeholder text.

The minimum gray we can use to be #777777, which is pretty dark gray. If that is what we need to do, let's do it. This is, in my opinion, another argument in favor of traditional field labels.

**Comment by pdpinch** [comment]
@roberthouse54 Is it just the placeholder text, or does it include the values in the select list?

**Comment by pdpinch** [comment]
I'm moving this back to "in design" since it's not ready to be worked on yet.

**Comment by roberthouse54** [comment]
I think this is mostly for the placeholder text, but DB and I are discussing this more broadly tomorrow
